package.json

```diff
@@ -2,9 +2,9 @@
   "name": "@no-day/fp-ts-graph",
   "version": "0.1.0-rc.2",
   "homepage": "https://github.com/no-day/fp-ts-graph",
-  "main": "dist/index.js",
+  "main": "dist/src/index.js",
```

Comment:
Good point, this should not be like this. 

However, it's "right" in the current setup. As you can see if you run the build script, both "src" and "examples" go into the `dist` folder.

We should change the "include" section in `tsconfig.json` to not include "examples". Basically: `"include": ["src/**/*.ts"]`. 

Then I'll accept the PR :)